SEED-platform/seed
# Fixed issue with overlapping meter readings with new ESPM format

#### Any background context you want to provide?
When we revised the code to accept the new ESPM format for monthly meter usage, there ended up being an issue where the data was being added twice (so 24 months of data instead of 12 months when uploading a year's worth).

#### What's this PR do?
Switches from checking whether the field starts with the header_string to checking whether the stripped field (removed the units and space at the end of the string) is equal to the header_string.

#### How should this be manually tested?
Upload the example file called "example-data-request-response-new-format.xlsx" (saved in seed/tests/data) and see whether it has 48 records for each fuel type or 96 records.

#### What are the relevant tickets?
#3744 

#### Screenshots (if appropriate)
